channels/mx.m3u

```diff
@@ -195,3 +195,5 @@ https://5ca3e84a76d30.streamlock.net/gpacifico3/obregon.smil/playlist.m3u8
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x3m1xfy
 #EXTINF:-1 tvg-id="UnoTV.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="",Uno TV (720p) [Not 24/7]
 https://ooyalahd2-f.akamaihd.net/i/UnoTV01_delivery@122640/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="",Canal 5
+ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/5mxt291
```

Comment:
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/5mxt291
```

I suggest not adding it, as they change their name almost every week.

Reply:
Yeah, name has already been changed